incubator/basic-auth/basic-auth/values.yaml

```diff
@@ -0,0 +1,53 @@
+# SLATE instance tag
+Instance: ''
+
+# SSSD Configuration
+# This is an entire file which maps directly to /etc/sssd/sssd.conf
+# TODO (More Info)
+sssd: |+
+  [sssd]
+  config_file_version = 2
+  reconnection_retries = 3
+  sbus_timeout = 30
+  services = nss, pam
+  domains = ad.utah.edu
+
+  [nss]
+  filter_groups = root
+  filter_users = root
+  reconnection_retries = 3
+
+  [pam]
+  reconnection_retries = 3
+
+  [domain/ad.utah.edu]
+  debug_level = 3
+  enumerate = false
+  cache_credentials = true
+  use_fully_qualified_names = false
+
+  auth_provider = krb5
+  chpass_provider = krb5
+  krb5_realm = AD.UTAH.EDU
+  krb5_server = kdc1.chpc.utah.edu:88,kdc2.chpc.utah.edu:88,kdc3.chpc.utah.edu:88,kdc4.chpc.utah.edu:88,kdc5.chpc.utah.edu:88
+  krb5_auth_timeout = 15
+  krb5_store_password_if_offline = true
+  krb5_ccname_template = FILE:%d/krb5cc_%U_XXXXXX
+  krb5_ccachedir = /tmp
```

Comment:
Just leaving a note on this, with Docker's default seccomp profile, containers can't use the `keyctl` syscall so Kerberos can't use the kernel keyring to store Kerberos tokens. This may be a security concern?